src/sqladmin-fetcher.ts

```diff
@@ -69,21 +69,32 @@ function cleanGaxiosConfig() {
 }
 
 export interface SQLAdminFetcherOptions {
-  loginAuth?: GoogleAuth;
+  auth?: GoogleAuth<AuthClient> | AuthClient;
```

Comment:
This property is unused, only `loginAuth` is used in the constructor.

Reply:
agreed, the constructor only needs a single `loginAuth` object that the other instances should extend from using `authClient`